parity/run.rs

```diff
@@ -902,6 +903,7 @@ enum RunningClientInner {
 	Full {
 		informant: Arc<Informant<FullNodeInformantData>>,
 		client: Arc<Client>,
+		service: Arc<ClientService>,
```

Comment:
Can we rename this to something more descriptive, like `snapshot_service`, `service` in itself could be anything. Or is there any particular reason you picked `service`?

Reply:
It was just because the variable was named `service`, but could change for sure to `client_service`